### PR TITLE
Adding character <v> to banner font 

### PR DESCRIPTION
**Pull request by 12345rams**
Adding character <v> to banner font 
*Describes your contribution*

### Referenced Issues
Adding character <v> to banner font
#579

### CODE REVIEW

**Comment by AnshumanMahato** [comment]
@12345rams, wait for issues to be assigned before making a PR. Read the guidelines, please. Though I am considering this one, for further PRs, make sure to get the assignment first. It helps us avoid conflicts

File: Fonts/banner/banner.h [hunk 1]
```diff
@@ -522,29 +522,36 @@ class Banner : public Fonts
 
     */
 
-    /*
-        char **u()
```
Comment (AnshumanMahato) [inline Fonts/banner/banner.h:526]: You removed the placeholder for `u`. Please revert that

File: Fonts/banner/banner.h [hunk 2]
```diff
@@ -522,29 +522,36 @@ class Banner : public Fonts
...
-        {
-            char **character = getCharGrid(ROWS,COLS);
+   char **v()
```
Comment (AnshumanMahato) [inline Fonts/banner/banner.h:525]: Multiple definitions for v. Resolve this.

File: Fonts/banner/banner.h [hunk 3]
```diff
@@ -522,29 +522,36 @@ class Banner : public Fonts
...
+{
+    char **character = getCharGrid(7, 3); // Assuming a 7x3 character grid
 
-            //Enter the character grid in ROWS X COLS
+    // Fill the character grid for inverted "v"
+    character[0][0] = character[0][6] = '#';
+    character[1][1] = character[1][5] = '#';
+    character[2][2] = character[2][4] = '#';
+    character[3][3] = '#';
```
Comment (AnshumanMahato) [inline Fonts/banner/banner.h:533]: While you are working at it, also ensure that it aligns with the rest. Something like this must not happen
image

keeping a few top rows empty should do the trick.

**Review by AnshumanMahato — CHANGES_REQUESTED** [review]

File: Fonts/banner/banner.h (hunk 1 above)
Comment (12345rams) [reply Fonts/banner/banner.h]: done

File: Fonts/banner/banner.h (hunk 2 above)
Comment (12345rams) [reply Fonts/banner/banner.h]: done

File: Fonts/banner/banner.h (hunk 3 above)
Comment (12345rams) [reply Fonts/banner/banner.h]: done

**Review by AnshumanMahato — APPROVED** [review]
everything works fine

**Pull request merged**